test/test-all.js

```diff
@@ -17,22 +17,11 @@ BPromise.config({
 const app = createApp();
 
 function getPdfTextContent(buffer) {
-  return new BPromise((resolve, reject) => {
-    const pdfParser = new PDFParser();
-    pdfParser.on('pdfParser_dataError', (err) => {
-      reject(err);
-    });
-    pdfParser.on('pdfParser_dataReady', () => {
-      resolve(pdfParser.getRawTextContent());
-    });
-
-    pdfParser.parseBuffer(buffer);
-  });
+  return pdf(buffer)
+    .then(data => data.text.replace(/\u00A0/g, ' ').replace(/\u00AD/g, '-'));
```

Comment:
Does this fix on Linux only or all platforms? Would be nice to be able to run these locally on a Mac

Reply:
Yes, the test pass just fine on my Mac.